content/blog/infrastructure-as-code-on-hpe-greenlake-using-terraform-–-part-2.md

```diff
@@ -0,0 +1,363 @@
+---
+title: "Infrastructure-as-Code on HPE GreenLake using Terraform – Part 2 "
+date: 2022-04-27T11:33:25.740Z
+author: Didier Lalli
+authorimage: /img/didier-lalli.png
+thumbnailimage: /img/terraformprovider.png
+tags:
+  - hpe-greenlake
+  - devops
+  - terraform
+  - open source
+---
+The process of managing and provisioning computer data centers through machine-readable definition files, otherwise known as Infrastructure-as-Code (IaC), offers many significant benefits. It helps to increase operational agility, simplify management, reduce errors, and save cost. In this second post, I’ll explore how to extract more of the benefits of using IaC on HPE GreenLake through the use of Terraform.
+
+## Let’s recap
+
+In [my first blog](https://developer.hpe.com/blog/infrastructure-as-code-on-hpe-greenlake-using-terraform/), I covered HPE GreenLake with its Private Cloud Service and showed how to get started with Terraform and the Terraform provider for HPE GreenLake. In this post, I will start with the same VM configuration created in Part 1 and show you how to tap into more advanced functionality that’s provided by Terraform and the HPE GreenLake provider. If you’re coming in to this just now, you might want to follow the steps shown in my Part 1 blog post, up to the point (Terraform ready to apply) where the VM DidierTest1 is created in HPE GreenLake, as illustrated below:
+
+![DidierTest1 VM as created in Part 1](/img/terraform-greenlake-part2-blog-picture1.png "DidierTest1 VM as created in Part 1")
```

Comment:
Didier, suggest you use this image in the first blog too. The image in the first blog is fuzzy.